docs/cluster_admin/installation.md

```diff
@@ -225,6 +225,14 @@ ARM64 developer builds can be installed like this:
     $ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-operator-arm64.yaml
     $ kubectl apply -f https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-cr-arm64.yaml
 
+### s390x developer builds
+
+s390x developer builds can be installed like this:
```

Comment:
nit: `installed like this` -> `in this way`

Reply:
I copied this directly from arm64, should i change it there as well?